casper-finality-proofs/src/test_engine/utils/setup.rs

```diff
@@ -36,6 +62,14 @@ pub fn init_tests() -> Vec<TestCase> {
         "../vendor/consensus-spec-tests/tests/mainnet/capella/epoch_processing/justification_and_finalization/pyspec_tests/".to_string(),
         true,
     ));
+    tests.push(TestCase::new(
+        TestWrappers::WrapperMainnet,
+        "./src/test_engine/tests/compute_shuffled_index/".to_string(),
+        true));
+    tests.push(TestCase::new(
+        TestWrappers::WrapperMinimal,
+        "./src/test_engine/tests/compute_shuffled_index/".to_string(),
+        true));
```

Comment:
Add WrapperComputeShuffledIndexMainnetConsensus and Minimal respectfully with paths to the vendor submodule

Reply:
You should have 2 separate folders for minimal and mainnet in test_engine/tests/..
With the current setup one of the tests should be failing, shouldn't it?